src/components/Dropdown/ButtonDropdown/index.tsx

```diff
@@ -85,11 +85,11 @@ const ButtonDropdown = <T extends {}>(props: Props<T>) => {
     if (buttonRef.current && listRef.current && isListOpen && listPosition === 'top') {
       const bounds = buttonRef.current.getBoundingClientRect();
       if (groupBy) {
-        listRef.current.style.setProperty('bottom', `${bounds.height}px`);
+        listRef.current.style.setProperty('top', `-${bounds.height}px`);
```

Comment:
@viniciusaa, I can see in the implementation video that the dropdown is still going down before it goes up. Could you please fix this? That's not the expected behavior, and it isn't pleasant to see.

Reply:
If this needs more than just changing the CSS variable, please add a prop for this behaior.